src/ansys/hps/client/jms/schema/task_definition_template.py

```diff
@@ -25,7 +25,17 @@
 
 from ansys.hps.client.common import BaseSchema, ObjectSchema
 
-from .task_definition import HpcResourcesSchema, SoftwareSchema, WorkerContextSchema
+from .task_definition import HpcResourcesSchema, WorkerContextSchema
+
+
+class AvailableSoftwareSchema(BaseSchema):
```

Comment:
Other schemas for templates are called `TemplateResourceRequirementsSchema`, `TemplateInputFileSchema`, `TemplateOutputFileSchema`, etc. Let's use the same naming.
```suggestion
class TemplateSoftwareSchema(BaseSchema):
```